docs/wiki/deployment/process-auditing.md

```diff
@@ -152,6 +152,16 @@ A sample socket_event log entry looks like this:
 
 If you would like to log UNIX domain sockets use the hidden flag: `--audit_allow_unix`. This will put considerable strain on the system as many default actions use domain sockets. You will also need to explicitly select the `socket` column from the `socket_events` table.
 
+The `success` column has been deprecated and replaced with `status`:
+| Status value | Description |
+|-|-|
+| failed | Definitely failed |
+| succeeded | Definitely succeeded |
+| in_progress | The `connect`()` syscall has been marked as "in progress" (EINPROGRESS) and osquery can't determine whether it will succeed or not. Reserved for non-blocking sockets. |
+| no_client | The `accept` or `accept4` syscall returned with EAGAIN since there were not incoming connections. Reserved for non-blocking sockets. |
+
+Failed socket operations are ignored by default, but it is possible to include them using the following flag: `--audit_allow_failed_socket_events=true`.
+
```

Comment:
Can you add another paragraph after this (or maybe a bulleted list) that includes the socket auditing related flags and what they do?